packages/radix-ui-themes/src/props/layout.props.ts

```diff
@@ -342,6 +342,22 @@ const layoutPropDefs = {
     customProperties: ['--grid-row-end'],
     responsive: true,
   },
+  /**
+   * Sets the CSS **grid-area** property.
+   * Supports CSS strings and responsive objects.
+   *
+   * @example
+   * gridArea="header"
+   *
+   * @link
+   * https://developer.mozilla.org/en-US/docs/Web/CSS/grid-area
+   */
+  gridArea: {
+    type: 'string',
+    className: 'rt-r-ga',
+    customProperties: ['--grid-area'],
+    responsive: true,
+  },
```

Comment:
Let's put this prop in the alphabetical order with other grid props

Reply:
Fixed